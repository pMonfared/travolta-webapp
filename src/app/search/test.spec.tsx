import * as React from 'react';
import renderer from 'react-test-renderer';

import Search from './page';

describe('Search', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Search />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
