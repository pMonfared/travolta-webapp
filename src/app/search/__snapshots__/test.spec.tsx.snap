// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot renders 1`] = `
<div
  className="flex min-h-screen flex-col items-center justify-between p-24"
>
  <h3>
    Movies
  </h3>
  <ul />
</div>
`;
